tools/scripts/seed-exams/add-nano-ids.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Script that will add `id` fields to the `questions`, `wrong_answers`, and
+ * `correct_answers` of an exam file where the `id` doesn't exist.
+ */
+import { readFileSync, writeFileSync } from 'fs';
+import { customAlphabet } from 'nanoid';
+import yaml from 'js-yaml';
+
+const newId = customAlphabet('1234567890abcdefghijklmnopqrstuvwxyz', 10);
+
+// Change this to the path of the file you want to add id's to
+const examPath = './exams/example-certification-exam.yml';
```

Comment:
Not necessarily the end of the world, but I feel like this would be better as a CLI arg or as a prompt or something - rather than a hard-coded value that has to be changed every time we want to seed a new exam.

Reply:
You're probably right @naomi-lgbt. I would prefer if we just go with it like this for now. This is not for seeding the exams, it's for adding `id`'s to the YAML files of the exams. So it only needs to run once per exam file.